projects/libaom/build.sh

```diff
@@ -35,17 +35,23 @@ $CC $CFLAGS -std=c99 -c \
   -I$WORK \
   $SRC/aom/common/tools_common.c -o $WORK/tools_common.o
 
-# build fuzzer
-fuzzer_name=av1_dec_fuzzer
+# build fuzzers
+fuzzer_modes=( serial threaded )
 
-$CXX $CXXFLAGS -std=c++11 \
-  -I$SRC/aom \
-  -I$WORK \
-  -Wl,--start-group \
-  -lFuzzingEngine \
-  $SRC/${fuzzer_name}.cc -o $OUT/${fuzzer_name} \
-  $WORK/libaom.a $WORK/ivfdec.o $WORK/tools_common.o \
-  -Wl,--end-group
+for mode in "${fuzzer_modes[@]}"; do
+  fuzzer_src_name=av1_dec_fuzzer
+  fuzzer_name=${fuzzer_src_name}_${mode}
```

Comment:
Now all fuzz targets will have _serial or _threaded in them. So, old reports will get closed since old fuzz target wont be found in new builds. Can you just have "_serial" as "" and use _threaded for _threaded ?